.gitpod.yml

```diff
@@ -0,0 +1,34 @@
+image:
+  file: .devcontainer/.gitpod.Dockerfile
+  context: .devcontainer
+
+tasks:
+  - init: >
+      git submodule init
+      && git submodule update
+      && ./autogen.sh
+      && ./configure --enable-ccache --enable-tracy --enable-tracy-capture --enable-tracy-gui
```

Comment:
Agreed, the extra debugging checks should be added to the default `./configure` in `.gitpod.yml` (if we ever do this).

I thought Tracy would be interesting to enable not because I expect people to be able to get representative performance analysis out of Gitpod, but because, as Graydon has pointed out, actually running and interacting with software components is often the best way of learning about them.  He described the cycle of ledger closes, for example, as the "heartbeat" of stellar-core, and I _think_ you could see that heartbeat in a trace of an instance running from Gitpod, even if the absolute numbers wouldn't mean much of anything.